### ISSUE TITLE
VEP.ipynb scoring

### ISSUE THREAD

**Issue by Anshullllllll** [songlab-cal/gpn#30]
Hii @gonzalobenegas i have some questions regarding vep.ipynb output

1. What is the threshold for scores so that we can conclude that a variant is pathogenic or not?
2. How scores are being calculated?
3. If i want to get probabilities values along with scores, then what changes i can do in script?
4. For insertion and multiple sequence variant (MSV), why script throw error?
for example:
![Screenshot from 2024-08-24 17-55-42](https://github.com/user-attachments/assets/b6ba731d-14ee-4216-9e72-a88dec264e70)

**Comment by gonzalobenegas** [comment]
Hello, thanks for your questions! 

> What is the threshold for scores so that we can conclude that a variant is pathogenic or not?
This is something we describe in the latest version of the paper. We don't recommend taking a binary cutoff, but interpret deleteriousness as a continuum. If you need a number, you could say -7.

> How scores are being calculated?
The log-likelihood ratio between the two alleles, when the human variant position is masked, conditioned on the rest of the MSA.

> If i want to get probabilities values along with scores, then what changes i can do in script?
I'd recommend running `python -m gpn.msa.inference logits`, and then transform the resulting logits into probabilities using `scipy.special.softmax`. 

> For insertion and multiple sequence variant (MSV), why script throw error?
None of the current models support scoring indels. For multiple sequence variants, it would work if you separate them into different rows (treat them as different variants, one per row).
